### ISSUE TITLE
[Campus DLC] Academic year gets really really long in Real time

### ISSUE THREAD

**Issue by ArcturusZhang** [dymanoid/RealTime#235]
In the first academic year, everything is fine. But in the second academic year, it is stretched to as long as two years, which is a really long time.
I have many mods, but those who interferes with time is only Real Time.
Could you please help me with this?

**Comment by ArcturusZhang** [comment]
Screen shot here
![Literally 8 years of an academic year](https://user-images.githubusercontent.com/10554446/66253816-695d0c00-e7a0-11e9-8ccb-60a71453a910.png)

**Comment by dymanoid** [comment]
Spotted the possible issue cause, continue investigating.

**Comment by ihatearial** [comment]
Is this related to another problem with matches at campus stadiums? The day arrives and the match is moved to another date. Also, the "original" match never happens.

**Comment by dymanoid** [comment]
@ihatearial No, this is something different. Which version of Real Time are you running? There were some issues with events in the previous versions of the mod.

**Comment by dymanoid** [comment]
@ArcturusZhang Currently I cannot reproduce the issue with the latest game version and the latest Real Time version. Could you please check whether you still have any issues?

**Comment by ArcturusZhang** [comment]
> @ArcturusZhang Currently I cannot reproduce the issue with the latest game version and the latest Real Time version. Could you please check whether you still have any issues?

Sure, thanks for the great work! But I am afraid I may not reproduce this either. To reproduce this may take a really long time in real time. I will let you know if this issue still exists. Thanks again!
